### ISSUE TITLE
Teensy 4.0/4.1 Support

### ISSUE THREAD

**Issue by tothambrus11** [luni64/TeensyStep#161]
Hi, thank you for this library!
Is there any known reason why the newer Teensy models are not supported?
I am about to buy one of those, and I could try them out and see if they work with the library / I could help fixing some issues with it.

Btw, great library & article, it was an interesting read :)

**Comment by cbradburne** [comment]
Yep, check out [TeensyStep4](https://github.com/luni64/TeensyStep4) 
It's still work in progress but it's been working well for me.
